lib/setup.ex

```diff
@@ -1,4 +1,6 @@
 defmodule Setup do
```

Comment:
Why is this module named `Setup`? What does `Setup` do? This definitely needs a better name.